package student_sergey_egorichev.lesson_11.level_3.task_21;

// 1 – т.к сначала будет перехвачен TestException, затем отработает finally
